lua/nvim-treesitter/parsers.lua

```diff
@@ -1510,9 +1510,9 @@ list.systemtap = {
 
 list.t32 = {
   install_info = {
-    url = "https://codeberg.org/xasc/tree-sitter-t32",
+    url = "https://gitlab.com/xasc/tree-sitter-t32",
     files = { "src/parser.c", "src/scanner.c" },
-    revision = "1dd98248b01e4a3933c1b85b58bab0875e2ba437",
+    revision = "9d2520ae9886d3a768a352ec80db8762afb5232d",
```

Comment:
It's pretty obvious that nvim-treesitter will still pick up changes for your repo, even if you set the revision. So I believe it'd be better to remove this.

This can be re-added back from user-config level, and I think that's better

Reply:
No, this is how a parser is locked from automatic updates on `master`.

Reply:
The latest of changes to the grammar was a bigger one. With these changes, the feature coverage of the grammar is quite complete, and I would expect things to slow down a bit now. The node names should also be quite stable.
I think I can continue without pinning the grammar to a certain version. At least I will try to be disciplined regarding changes that break things here.

Reply:
Then we can indeed drop the field from here on out (follow-up PR welcome!)

Reply:
> No, this is how a parser is locked from automatic updates on `master`.

I didn't check why it keeps showing in parser update list by the github-bot, looking into it and saw it was because of codeberg :smile:. Shouldn't be a problem anymore

So I take all the blame here